units/Incendiary_Drone.cfg

```diff
@@ -0,0 +1,119 @@
+#textdomain wesnoth-loti
+[unit_type]
+    id=Incendiary Drone
```

Comment:
Can you use `base_unit` here?

Reply:
Probably.  I don't know what I'm doing here, so I probably just copied the file.